src/payment/payment-request-transformer.ts

```diff
@@ -0,0 +1,90 @@
+import { pick } from 'lodash';
+
+import { mapToInternalAddress } from '../address';
+import { mapToInternalCart } from '../cart';
+import InternalCheckoutSelectors from '../checkout/internal-checkout-selectors';
+import { StandardError } from '../common/error/errors';
+import { mapToInternalCustomer } from '../customer';
+import { mapToInternalOrder } from '../order';
+import { mapToInternalShippingOption } from '../shipping';
+
+import isVaultedInstrument from './is-vaulted-instrument';
+import Payment from './payment';
+import PaymentMethod from './payment-method';
+import PaymentMethodSelector from './payment-method-selector';
+import PaymentRequestBody from './payment-request-body';
+
+export default class PaymentRequestTransformer {
+    static transform(payment: Payment, checkoutState: InternalCheckoutSelectors): PaymentRequestBody {
```

Comment:
We prefer not to use `static` methods as it's not possible to mock them. Can you make this an instance method and inject the instance into places where you need it?